### PR TITLE
 Use MKL-DNN 0.16 on Travis-CI and AppVeyor

### PR DESCRIPTION
**Pull request by msakai**
Please merge this PR after #86.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -26,11 +26,11 @@ install:
 
 - if [%TARGET%]==[mingw] (
     pacman -S --needed --noconfirm mingw-w64-x86_64-toolchain mingw-w64-x86_64-cmake mingw-w64-x86_64-opencv mingw-w64-x86_64-protobuf mingw-w64-x86_64-python3 mingw-w64-x86_64-python3-pip mingw-w64-x86_64-python3-numpy mingw-w64-x86_64-hdf5 mingw-w64-x86_64-ca-certificates &&
-    curl -omingw-w64-x86_64-mkl-dnn-0.15-1-x86_64.pkg.tar.xz -L https://github.com/pfnet-research/menoh/releases/download/v1.0.3/mingw-w64-x86_64-mkl-dnn-0.15-1-x86_64.pkg.tar.xz &&
-    pacman -U --noconfirm mingw-w64-x86_64-mkl-dnn-0.15-1-x86_64.pkg.tar.xz
+    curl -omingw-w64-x86_64-mkl-dnn-0.16-1-x86_64.pkg.tar.xz -L https://github.com/pfnet-research/menoh/releases/download/v1.0.3/mingw-w64-x86_64-mkl-dnn-0.16-1-x86_64.pkg.tar.xz &&
```
Comment (okdshin) [inline appveyor.yml]: Version string "0.16" is embedded in script. How about to make it variable as in .travis.yml?

Comment (msakai) [reply appveyor.yml]: I added another commit to use environmental variables instead of hard coded version numbers.

**Comment by msakai** [comment]
@okdshin I'll merge it if there are no other problems.

**Comment by okdshin** [comment]
No problem. Please.

**Comment by msakai** [comment]
Thanks!

**Pull request merged**